integration/tools/validation/src/main/java/alluxio/fuse/correctness/Options.java

```diff
@@ -0,0 +1,214 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse.correctness;
+
+import com.google.common.base.Preconditions;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.ParseException;
+
+/**
+ * This class stores the options passed in from CLI.
+ */
+public final class Options {
+  public static final CommandLineParser PARSER = new DefaultParser();
+  private final String mTmpDir;
+  private final String mFuseDir;
+  private final IOOperation mOperation;
+  private final int mNumClients;
+  private final int mNumFiles;
+  private final int mTimeout;
+
+  /**
+   * Constructs a {@Options} object through CLI.
+   *
+   * @param args arguments from CLI
+   * @return a {@Options} holding all the test options
+   */
+  public static Options createOptions(String[] args) {
+    CommandLine cli;
+    try {
+      cli = PARSER.parse(CorrectnessOptionsParser.OPTIONS, args);
+    } catch (ParseException e) {
+      throw new RuntimeException("Error parsing command line input.", e);
+    }
+    String fuseDir = cli.getOptionValue(CorrectnessOptionsParser.FUSE_DIR_OPTION_NAME);
+    String localDir = cli.hasOption(CorrectnessOptionsParser.TMP_DIR_OPTION_NAME)
+        ? cli.getOptionValue(CorrectnessOptionsParser.TMP_DIR_OPTION_NAME)
+        : Constants.DEFAULT_LOCAL_DIR;
+    IOOperation operation = IOOperation.fromString(
+        cli.hasOption(CorrectnessOptionsParser.TEST_OPERATION_OPTION_NAME)
+            ? cli.getOptionValue(CorrectnessOptionsParser.TEST_OPERATION_OPTION_NAME)
+            : Constants.DEFAULT_OPERATION);
+    int numThreads = Integer.parseInt(
+        cli.hasOption(CorrectnessOptionsParser.CLIENT_NUMBER_OPTION_NAME)
+            ? cli.getOptionValue(CorrectnessOptionsParser.CLIENT_NUMBER_OPTION_NAME)
+            : Constants.DEFAULT_NUM_THREADS);
+    int numFiles = Integer.parseInt(
+        cli.hasOption(CorrectnessOptionsParser.FILE_NUMBER_OPTION_NAME)
+            ? cli.getOptionValue(CorrectnessOptionsParser.FILE_NUMBER_OPTION_NAME)
+            : Constants.DEFAULT_NUM_FILES);
```

Comment:
For all the constants shared by different classes, better to put in a separate class but if it's only be used once by a single class, can be put in this class as a static variable